parser/javascript/meriyah.ts

```diff
@@ -21,6 +21,7 @@ export const meriyah: Parser<typeof Meriyah, Options> = {
     defaultValue: {
       module: true,
       ranges: true,
+      raw: true,
```

Comment:
Since other ES parsers add `raw` to `Literal` by default, hope this is fine to enable by default.